Docker update (#128)
cybernetisk/internsystem-v2: CONTRIBUTING.md

This is further described in the [Getting Started](#getting-started) section

## Getting started
Note that you'll probably only need to do these first steps once, so if you don't understand what's happening and what the commands are doing, no worries and if you have questions about something, or issues setting up your environment, don't hesitate to ask.

### Forking the repo
To get started contributing, make a fork of the cyb repo. If you're new to git and GitHub, forking is a feature in GitHub, where you make your own personal copy of a repo. In your own fork you have both read and write access and you can make whatever changes you want without affecting cyb's repo. To get your changes applied to cyb's repo you need to make a pull-request, but more on that later.

### Making a local clone of your fork
When you're done making the fork, you can "download" or clone the repo to your local machine. This can be done though the git CLI (Command Line Interface) with the following command:

```
git clone <url-to-your-fork>
```

> This url will typically look like: `https://github.com/<your_username>/internsystem-v2.git` or `git@github.com:Sebbben/internsystem-v2.git` if you have set up ssh-keys for you github (which is recomended).

Reviewer: ```suggestion
This URL will typically look like: `https://github.com/<your_username>/internsystem-v2.git` or `git@github.com:<your_username>/internsystem-v2.git` if you have set up SSH keys on GitHub (which is recommended).
```

> When you have done this, a directory named "internsystem-v2" will be created on you computer containing the source-code for the project. 

Reviewer: ```suggestion
When you have done this, a directory named `internsystem-v2` will be created on your computer containing the source-code for the project. 
```

#### You have now created a feature branch!
You have now created a feature branch. While working on the code for the feature, remember to commit often and provide descriptive and short commit messages for the changes made in the commit. When you want to upload the code changes to your GitHub repo, just run `git push` and the code will be sent on its way. This is not the same as making a pull-request tho. See [Creating a pull-request](#creating-a-pull-request) for more info on publishing your code to the cyb repo.

### Running the site

> To make running the website as frictionless as possible we have created a docker setup with a `docker-compose` file. This makes it so that all you need to do to get the site running along with a database is to run `docker compose up -d` and thats it. Running this should start both a database (or run the container from previous runs of the docker-compose file) and then the website. When starting the website docker will first run commands to initialize the database and then populate it with some nececary data to get the website working.

Reviewer: ```suggestion
To make running the website as easy as possible we have created a docker setup with a `docker-compose` file. All you need to do to get the site running along with a database is to run `docker compose up -d`. This command should both start a database (or run the container from previous runs of the `docker-compose` file) and then the website. When starting the website, docker will first run commands to initialize the database, and then populate it with some necessary data to get the website working.
```

> After you have run this command the website should be avaliable by navigating to `http://localhost:3005`

Reviewer: ```suggestion
After you have run this command the website should be available by navigating to `http://localhost:3005`
```
